cf-java-logging-support-core/src/main/java/com/sap/hcp/cf/logging/common/converter/LineWriter.java

```diff
@@ -26,7 +24,7 @@ public List<String> getLines() {
      */
     @Override
     public void write(String str, int off, int len) {
-        if (StringUtils.isNotBlank(str)) {
+        if (str != null && !str.isBlank()) {

```

Comment:
any reason for not providing a shared method in `support-core` module to be used everywhere?